### PR TITLE
Printusers

### PR DESCRIPTION
**Pull request by LinusUnghammar**
This PR will 
- print all users existing in db
- create addfriend resource 

this will close down #54

### CODE REVIEW

File: backend/database/migrations/2021_05_18_070618_create_addfriends_table.php [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::dropIfExists('addfriends');
+    }
+}
```
Comment (jagestedt) [inline backend/database/migrations/2021_05_18_070618_create_addfriends_table.php:34]: add newline

**Review by jagestedt — CHANGES_REQUESTED** [review]

File: backend/database/migrations/2021_05_18_070618_create_addfriends_table.php (hunk 1 above)
Comment (frankemon) [reply backend/database/migrations/2021_05_18_070618_create_addfriends_table.php:34]: was this resolved? cant see atm

Comment (jagestedt) [reply backend/database/migrations/2021_05_18_070618_create_addfriends_table.php]: i still see it: 
image

Comment (LinusUnghammar) [reply backend/database/migrations/2021_05_18_070618_create_addfriends_table.php]: @ichbinjaeger good catch, it's fixed

**Comment by frankemon** [comment]
Yeah go ahead and merge it, looks fine

**Review by frankemon — APPROVED** [review]
Nice

**Review by jagestedt — APPROVED** [review]

**Pull request merged**